woocommerce/payment-gateway/assets/js/frontend/sv-wc-payment-gateway-my-payment-methods.coffee

```diff
@@ -61,6 +64,30 @@ jQuery( document ).ready ($) ->
 				event.preventDefault() if $( this ).hasClass( 'disabled' )
 
 
+		# Replace Method column content with Title column content, for FW tokens.
+		#
+		# @since 5.6.0-dev
+		replace_method_column: =>
+
+			$( '.woocommerce-MyAccount-paymentMethods' ).find( 'tr' ).each ( index, element ) =>
+
+				# delete the Title header
+				$( element ).find( 'th.woocommerce-PaymentMethod--title' ).remove()
+
+				titleColumn = $( element ).find( 'td.woocommerce-PaymentMethod--title' )
+
+				# Title column is not empty, this is a FW token
+				if ( titleColumn.children().length > 0 )
+
+					# replace Method column
+					$( element ).find( 'td.woocommerce-PaymentMethod--method' ).replaceWith( titleColumn )
```

Comment:
I noticed that with that replacing the column causes the mobile version of the table to show **Title** instead of **Method** because of the data attribute in the Title column: https://cloud.skyver.ge/7KuR2yln

Additionally, I believe any CSS rules for `.woocommerce-PaymentMethod--method` would not apply anymore to cells in framework rows.

Reply:
Yes, good idea, I've changed it so it only replaces the content.